### DISCUSSION TITLE
When will Storefront API be released?

### DISCUSSION THREAD

**Discussion by waney** [lunarphp/lunar#1710] [Q&A] [answered]
Thank you.

**Comment by adevade** [comment, marked as answer]
It's on the roadmap for v1.x, so probably earliest v1.1 since v1.0 has it's own milestone. No dates are set yet, but v1.0-beta should be coming up in the next couple of weeks! 👍

https://github.com/orgs/lunarphp/projects/8/views/15?pane=issue&itemId=48283604

**Comment by waney** [comment]
Thanks you and look forward to it.
